update changelog for v0.2.0 (#208)
open-mmlab/mmocr: docs/changelog.md

# Changelog

## v0.2.0 (18/5/2021)

**Highlights**

> 1. Support Bert-softmax (NAACL'2019)

Reviewer: ```suggestion
1. Add the NER approach Bert-softmax (NAACL'2019)
```

> 2. Support DRRG (CVPR'2020)

Reviewer: ```suggestion
2. Add the text detection method DRRG (CVPR'2020)
```

> 3. Support FCENet (CVPR'2021)

Reviewer: ```suggestion
3. Add the text detection method FCENet (CVPR'2021)
```

> 4. Add text detection and recognition end-to-end demo.

Reviewer: ```suggestion
4.  increase the ease of use via adding text detection and recognition end-to-end demo, and colab online demo as well as simplifying the installation.
```

- Add end-to-end demo [#105](https://github.com/open-mmlab/mmocr/pull/105)
- Support batch inference [#86](https://github.com/open-mmlab/mmocr/pull/86) [#87](https://github.com/open-mmlab/mmocr/pull/87) [#178](https://github.com/open-mmlab/mmocr/pull/178)
- Add TPS preprocessor for text recognition [#117](https://github.com/open-mmlab/mmocr/pull/117) [#135](https://github.com/open-mmlab/mmocr/pull/135)
- Add demo documentation [#151](https://github.com/open-mmlab/mmocr/pull/151) [#166](https://github.com/open-mmlab/mmocr/pull/166) [#168](https://github.com/open-mmlab/mmocr/pull/168) [#170](https://github.com/open-mmlab/mmocr/pull/170) [#171](https://github.com/open-mmlab/mmocr/pull/171)
- Add checkpoint for Chinese recognition [#156](https://github.com/open-mmlab/mmocr/pull/156)
- Add metafile [#175](https://github.com/open-mmlab/mmocr/pull/175) [#176](https://github.com/open-mmlab/mmocr/pull/176) [#177](https://github.com/open-mmlab/mmocr/pull/177) [#182](https://github.com/open-mmlab/mmocr/pull/182) [#183](https://github.com/open-mmlab/mmocr/pull/183)
- Add support for numpy array inference [#74](https://github.com/open-mmlab/mmocr/pull/74)

**Bug Fixes**

> - Fix duplicated points due to transform for textsnake [#130](https://github.com/open-mmlab/mmocr/pull/130)

Reviewer: ```suggestion
- Fix the duplicated point bug due to transform for textsnake [#130](https://github.com/open-mmlab/mmocr/pull/130)
```

- Fix CTC loss NaN [#159](https://github.com/open-mmlab/mmocr/pull/159)
- Fix error raised if result is empty in demo [#144](https://github.com/open-mmlab/mmocr/pull/141)

> - Fix results missed if a large number of boxes on one image [#98](https://github.com/open-mmlab/mmocr/pull/98)

Reviewer: ```suggestion
- Fix results missing if one image has a large number of boxes [#98](https://github.com/open-mmlab/mmocr/pull/98)
```

> - Fix package missed in dockerfile [#109](https://github.com/open-mmlab/mmocr/pull/109)

Reviewer: ```suggestion
- Fix package missing in dockerfile [#109](https://github.com/open-mmlab/mmocr/pull/109)
```

**Improvements**

> - Simplify installation procedure [#188](https://github.com/open-mmlab/mmocr/pull/188)

Reviewer: ```suggestion
- Simplify installation procedure via removing compiling [#188](https://github.com/open-mmlab/mmocr/pull/188)
- Speed up panet post processing so that it can detect dense texts [#188](https://github.com/open-mmlab/mmocr/pull/188)
```
